pandas/io/html.py

```diff
@@ -716,9 +713,9 @@ def _validate_flavor(flavor):
     return flavor
 
 
-def _parse(flavor, io, match, header, index_col, skiprows,
-           parse_dates, tupleize_cols, thousands, attrs, encoding,
-           decimal):
+def _parse(flavor, io, match,
+           attrs, encoding,
+           **kwargs):
```

Comment:
Can you put those on one line? (not needed for PEP8 to put on multiple lines here)